django/utils/http.py

```diff
@@ -110,16 +110,18 @@ def urlencode(query, doseq=False):
 
 def cookie_date(epoch_seconds=None):
     """
-    Format the time to ensure compatibility with Netscape's cookie standard.
+    Format the time
 
     `epoch_seconds` is a floating point number expressed in seconds since the
     epoch, in UTC - such as that outputted by time.time(). If set to None, it
     defaults to the current time.
 
-    Output a string in the format 'Wdy, DD-Mon-YYYY HH:MM:SS GMT'.
+    Output a string in the format 'Wdy, DD Mon YYYY HH:MM:SS GMT'.
     """
-    rfcdate = formatdate(epoch_seconds)
-    return '%s-%s-%s GMT' % (rfcdate[:7], rfcdate[8:11], rfcdate[12:25])
+    return formatdate(
```

Comment:
Single line looks fine here. For consistency with other code, I'd pass`epoch_seconds` as an arg.

Reply:
You mean like this?
```py
formatdate(epoch_seconds, usegmt=True)
```

Reply:
Yes